docs/user-documentation/moderne-platform/how-to-guides/reporting.md

```diff
@@ -0,0 +1,61 @@
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Reporting
+
+## Overview
+
+Moderne offers some reports:
+
+* **Usage by organization** - every recipe run and associated commit by every user for a specific organization. 
+
+## Usage by organization
+The generated report includes every recipe run and associated commit for a given organization. This report includes
+the recipe name, time saved, user that ran it, recipe, as well as various statistics of the recipe run. 
+If any commit was created in association with the recipe run details of the commit are included: the type of 
+commit, status of the commit job, as well as other statistics of the commit job.
+
+The usage report is accessible from `https://<TENANT>.moderne.io/devcenter/{organization}`
+
+These reports can be downloaded using the download button: ![](./assets/usage-report-download.png)
```

Comment:
Can we get a higher quality photo for this? It seems quite blurry to me. Maybe we can zoom in on the browser?

Reply:
Let me make a new one, I think this was my fault because I was doing some resizing